javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var outPut = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] !== null) {
+      outPut.push(values[i]);
+    } else {
+      continue;
+    }
```

Comment:
This can be removed, the loop will just continue if the condition above isn't met anyway